src/main/java/uk/co/ramp/covid/simulation/population/Person.java

```diff
@@ -108,20 +51,24 @@ public void setRecovered(boolean recovered) {
         this.recovered = recovered;
     }
 
-    public int getMIndex() {
-        return this.mIndex;
+    public CommunalPlace getPrimaryCommunalPlace() {
+        return primaryPlace;
```

Comment:
I would prefer to use 'this.primaryPlace'

Reply:
Sure I can make this change.